### PR TITLE
checkbox styling

### LINKED ISSUE #1167: Checkboxes Not Styled
**Problem**
The checkboxes in new UI don't have any styling. Needs to be done at all pages. 

**Are your working on the issue**
No

### PR DESCRIPTION
**Pull request by abstrekt**
Fixes #1167

#### Describe the changes you have made in this PR -
added styling to the checkbox, awaiting review to proceed to all checkbox.
### Screenshots of the changes (If any) -

CircuitVerse - Digital Circuit Simulator

Firefox_Screenshot_2020-03-20T12-18-13 714Z

### CODE REVIEW

**Comment by abstrekt** [comment]
@nitin10s could you please review if this is acceptable, if yes also check if the checkbox is really working, thanks.

**Comment by abstrekt** [comment]
> 
> 
> @abstrekt this check box is looking good . Please look for other checkboxes present in the codebase such as featured circuit checkbox in admin panel and others if available. Also please fix the codeclimate issues 👍.

Yes, I'm awaiting a review from Nitin before I proceed, thanks for your review too.

**Comment by yash15601** [comment]
@abstrekt also change it in edit profile page

File: app/assets/stylesheets/signup.scss [hunk 1]
```diff
@@ -84,4 +84,51 @@ select {
...
+  &:checked {
+    ~ {
+      .checkmark {
+        background-color: $green;
+      }
+    }
+  }
+}
+
+input[type=checkbox]:checked ~ .checkmark::after {
```
[the remark this answers is not in the document]
Comment (abstrekt) [reply app/assets/stylesheets/signup.scss]: :(

**Comment by abstrekt** [comment]
@nitin10s

**Comment by abstrekt** [comment]
Sorry for the mess, was learning some git commands. Will create a new PR for this.

**Pull request closed without merging**